hack/toolchain.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-K8S_VERSION="${K8S_VERSION:="1.29.x"}"
+K8S_VERSION="${K8S_VERSION:="1.30.x"}"
```

Comment:
Let's keep default at 1.29.x for now
```suggestion
K8S_VERSION="${K8S_VERSION:="1.29.x"}"
```